cpp/src/groupby/sort/group_argmin.cu

```diff
@@ -40,24 +40,23 @@ std::unique_ptr<column> group_argmin(column_view const& values,
                                  num_groups,
                                  group_labels,
                                  stream,
-                                 mr);
+                                 cudf::get_current_device_resource_ref());
```

Comment:
Similar to this needs to be applied for `indices` in `group_argmax.cu`.

Reply:
Oh wait, no. Since we are moving the null mask to the output, we need to use `mr` here instead.

Reply:
That is true.